include/lsst/meas/base/PixelFlags.h

```diff
@@ -74,8 +80,18 @@ class PixelFlagsAlgorithm : public SimpleAlgorithm {
         CR,
         CR_CENTER,
         BAD,
+        CLIPPED,
         N_FLAGS
     };
+    
+    afw::table::Key<afw::table::Flag> _generalFailure;
+    afw::table::Key<afw::table::Flag> _offImageKey;
+    typedef std::map<std::string, afw::table::Key<afw::table::Flag> > KeyMap;
+    KeyMap _predefinedAnyKeys;
+    KeyMap _predefinedCenterKeys;
+    KeyMap _centerKeys;
+    KeyMap _anyKeys;
+
```

Comment:
Let me think about this further, but the first reason that comes to mind, is that as of now, all the predefined keys are expected to be present in so many places in the code they have to be there, if they were in the same config structure, it may be possible for people to remove needed keys, breaking the stack. I agree ideally that we would have there be no explicit dependence on specific keys, but that would need to wait until the full flaghandler refactoring